claat/main.go

```diff
@@ -77,6 +77,7 @@ func main() {
 			GlobalGA:  *globalGA,
 			Output:    *output,
 			Prefix:    *prefix,
+			Srcs:      flag.Args(),
```

Comment:
Do you need to store the srcs here for some other dependency? I do like the var change from args to srcs but why not just use:

srcs = unique(flag.args())

and then use srcs from that point forward but continue using a temp variable.